GTHC/web
# Availability Labels to Shift Users

## Description

Implemented the availability functionality to adding users to a shift when creating or updating a shift. The availability of a user is calculated by their lowest possible availability within a range of a starting and ending time of a shift. 

Each level of availability is represented by the colors: Red (Unavailable), Yellow (Somewhat), and Green (Available). 

## Motivation and Context


This is a continuation of #39 and it is actually implementing the information that users give to their team about their weekly availability to the process of creating a shift.

## How Has This Been Tested?
**Remember:** _Always migrate your database in case of changes (yarn dev:db:migrate)_



1. New seeds have been created to give other users randomized availabilities, so do the following:
   a. Run `yarn flush`, and respond 'y'
   b. Good to go! Run 'yarn dev:start`
2. Create a user, and join a team.
3. Create a shift, and add users. See that the dropdown now has users aprox. availability labeled.
4. Update a shift, and add/remove users. Check the dropdown.

## Screenshots (if appropriate):
![image](https://user-images.githubusercontent.com/13587407/50426501-2eb36400-085d-11e9-8c6e-d85c42633ee6.png)


## Types of changes


## Checklist:
